youtube_dl/downloader/external.py

```diff
@@ -181,7 +181,8 @@ def _call_downloader(self, tmpfilename, info_dict):
 
         args = [ffpp.executable, '-y']
 
-        args += self._configuration_args()
+        if self.exe == self.get_basename():
```

Comment:
You did not get it. We **should not** allow situations when downloader is two-fold. It should be either explicit one or selected not both at the same time.